src/interp/interp.cc

```diff
@@ -1691,36 +1698,66 @@ RunResult Thread::StepInternal(Trap::Ptr* out_trap) {
     case O::F32X4Add:          return DoSimdBinop(Add<f32>);
     case O::F32X4Sub:          return DoSimdBinop(Sub<f32>);
     case O::F32X4Mul:          return DoSimdBinop(Mul<f32>);
-    case O::F32X4Div:          return DoSimdBinop(FloatDiv<f32>);
-    case O::F32X4Min:          return DoSimdBinop(FloatMin<f32>);
-    case O::F32X4Max:          return DoSimdBinop(FloatMax<f32>);
+    case O::F32X4Div:
+      return DoSimdBinop(FloatDiv<f32>);
```

Comment:
Why not keep this on a single line like the others?  It doesn't looks like we are near the line limit.

Reply:
This was done by clang-format and enforced by the linting, if I'm not mistaken.

Reply:
This seems unfortunate.. I guess we should exclude this whole block from clang-formatting.